src/Routes.js

```diff
@@ -38,6 +39,9 @@ export default function Routes() {
           <Route exact path="/benefit">
             <Benefit />
           </Route>
+          <Route path = "">
```

Comment:
Hi @Aaishpra . I found a solution for this. It's just the matter of placing the code of the error page route within the switch options. It works fine once I removed the path and placed this error page route at the very bottom of the switch options. You can use this method on your code.

<img width="513" alt="Screen Shot 2020-10-28 at 11 44 50 am" src="https://user-images.githubusercontent.com/29667122/97377026-19a13380-1913-11eb-8f26-a48e40cb15a3.png">

![ezgif com-gif-maker (16)](https://user-images.githubusercontent.com/29667122/97376957-de9f0000-1912-11eb-8ce6-98d7e408242d.gif)

Reply:
Done